### ISSUE TITLE
Hibernate Metamodel Generator: duplicate class

### ISSUE THREAD

**Issue by mirkoliver** [johndevs/gradle-vaadin-plugin#505]
In my project I have hibernate metamodel classes. When using the task `vaadinCommit`, the following error is thrown - the metamodel classes seem to be generated two times!? How can I fix this?

`Using Gradle Vaadin Plugin 1.3.1
:vaadinPluginVersionCheck SKIPPED
:compileJavaNote: Hibernate JPA 2 Static-Metamodel Generator 5.2.12.Final
C:\MyFolder\EclipseWorkspace\myProject\build\classes\java\main\com\mydomain\myapp\db\bean\MyBean_.java:9: error: duplicate class: com.mydomain.myapp.db.bean.MyBean_
public abstract class MyBean_ {`

**Comment by johndevs** [comment]
Hi, the plugin does not have a ``vaadinCommit`` task, so that must be something custom to your project?

**Comment by mirkoliver** [comment]
Oh, sorry, I didnt't mean `vaadinCommit`, I've meant `vaadinCompile`

**Comment by johndevs** [comment]
It might be that your classpath has duplicate dependencies. Hard to say without the ``build.gradle`` file or the output of ``gradle dependencies``.

**Comment by mirkoliver** [comment]
Thank you!

I've found out, what has been wrong:
In my `depenedencies.gradle` I had added the `hibernate-jpamodelgen-5.2.12.Final.jar`, which generates the metamodel classes. But before generating them the second time, they are not deleted. So I've removed this dependency and the error is not anymore thrown.

Additional information: How to generate the hibernate metamodel classes?
Eclipse does the job for me: In my Eclipse project I have to add `hibernate-jpamodelgen-5.2.12.Final.jar`  manually to the classpath and then go to the "Annotation Processing" properties, "Enable annotation processing" option and configure the directory for the generated sources.
